### DISCUSSION TITLE
FactureX and Zugferd with Dompdf?

### DISCUSSION THREAD

**Discussion by kimler01** [dompdf/dompdf#3517] [General]
We will soon have new e-invoicing standards FactureX and Zugferd in Europe. I really like using DOMPDF to export my invoices. Is it possible to export the XML standard FactureX and Zugferd when the PDF is created? Does anyone have a solution

**Comment by bsweeney** [comment]
There is some effort to improve PDF/A support, see #3269. And the necessary XML embed can be generated using [using zugferd-php](https://github.com/dompdf/dompdf/issues/3443#issuecomment-2066845448). I haven't had a chance to fully review the spec to see if additional work would be needed.

**Comment by wblessen** [comment]
Have a look at https://github.com/dompdf/dompdf/issues/3534
